<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Home Dashboard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>881</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QPushButton" name="lightToggleButton">
       <property name="text">
        <string>Turn On</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="chartLayout"/>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="refreshButton">
       <property name="text">
        <string>Refresh Data</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSlider" name="brightnessSlider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QTableWidget" name="logTable">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="columnCount">
        <number>3</number>
       </property>
       <column>
        <property name="text">
         <string>Time</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Type</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Value</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="tempLabel">
       <property name="text">
        <string>0 °C</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="humidityLabel">
       <property name="text">
        <string>0 °C</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>915</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
